packages/plugin-api-logs/src/server/hooks/getFieldChange.ts

```
@@ -104,11 +107,19 @@ export async function getChanged(ctx): Promise<{ changed?: string[]; data?: any;
         continue;
       }
     }
+    const info = await repo.findOne({
+      filterByTk,
+      appends: [...appendSet],
+    });
+    const another = await repo.findOne({
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)